lib/Exception/AbstractException.ts

```diff
@@ -1,15 +1,18 @@
+import Request from '../Requester/Request';
 import { ResponseInterface } from '../Requester/Response/ResponseInterface';
 
 export default
 class AbstractException extends Error {
     private readonly _response: ResponseInterface;
+    private readonly _request: Request;
 
     /**
      * Constructor.
      */
-    constructor(response: ResponseInterface | undefined, message?: string) {
+    constructor(response: ResponseInterface | undefined, request: Request | undefined, message?: string) {
```

Comment:
This is a breaking change. Please rewrite the method signature (with overloads) to avoid this.